src/main/kotlin/racing/view/RacingUI.kt

```diff
@@ -1,16 +1,19 @@
-package racing
+package racing.view
+
+import racing.domain.RacingCars
+import racing.domain.Winners
 
 object RacingUI {
 
-    fun drawCars(cars: List<Car>) {
-        cars.forEach { car ->
+    fun drawCars(racingCars: RacingCars) {
+        racingCars.cars.forEach { car ->
```

Comment:
```suggestion
        racingCars.each { name, position ->
```
이런 함수를 만들어 보는건 어떨까요? 아래 댓글과 같은 맥락입니다. `RacingCars` 객체에게 메시지를 보내서 일을 시켜봅시다.

> RacingUI 입장에서 Winner 객체가 List<Car>를 가지고 있다는 정보를 몰라도 된다고 생각합니다.